cowprotocol/cowswap
# Add back 🎃 Halloween

# Summary

I just brought Halloween back :) 

@fairlighteth  only thing left is to change the logo.

# To Test

1. set dark mode
2. observe the spider web
3. Trade, and hear the diabolic laughter & visual effects
